src/index.html

```diff
@@ -7,8 +7,59 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="attr-req-value">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header href="#" class="header">
+      <a class="header__logo">
+        <img
+          src="/src/images/logo.png"
+          alt="logo"
+          class="header__img">
```

Comment:
<img width="1073" alt="Pasted Graphic" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/3709b95b-3f2a-469d-ba22-40929e53f3bc">
```suggestion
        <img
          src="/src/images/logo.png"
          alt="logo"
          class="header__img"
        >
```